build.gradle.kts

```diff
@@ -102,6 +103,12 @@ subprojects {
     apply {
         plugin("module")
     }
+
+//    afterEvaluate {
```

Comment:
Please review these changes and explain how n a comment whether they are persistent, or one-time.